src/types/billing.ts

```diff
@@ -7,17 +7,29 @@ import {
   Region as GenRegion,
   History as GenHistory,
   BillingDate as GenBillingDate,
+  ZuoraParams,
 } from 'src/client/unityRoutes'
 export {
   BillingContact,
-  CreditCardParams,
   Invoice,
   Invoices,
   PaymentMethod,
   PaymentMethods,
+  ZuoraParams,
```

Comment:
We're still going to need CreditCardParams here since it's required as part of the yml definition